Decred Wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -99,7 +100,7 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let seedWordCell = tableView.dequeueReusableCell(withIdentifier: "seedWordCell", for: indexPath) as! RecoveryWalletSeedWordCell
         
-        seedWordCell.lbSeedWordNum.text = "Word #\(indexPath.row + 1)"
+        seedWordCell.lbSeedWordNum.text = String(format: LocalizedStrings.word, indexPath.row + 1)
```

Comment:
Pfft. Missed the `#` in the original text. This would make more sense as `wordNumber`